### PR TITLE
docs: Update components version for Mender 3.7.8 release

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
FYI @estenberg

File: 07.Server-installation/04.Production-installation-with-kubernetes/03.Mender-server/docs.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ useradm:
 $(cat useradm.key | sed -e 's/^/      /g')
 EOF
 
-helm upgrade --install mender mender/mender -f mender-3.7.7.yml
+helm upgrade --install mender mender/mender -f mender-3.7.8.yml
```
Comment (oldgiova) [inline 07.Server-installation/04.Production-installation-with-kubernetes/03.Mender-server/docs.md:113]:
```suggestion
helm upgrade --install mender mender/mender --version 5.11.2 -f mender-3.7.8.yml
```
Could you please add the helm version as well? The upcoming latest will not be compatible with the 3.7 anymore.
Ref: https://github.com/mendersoftware/mender-docs/pull/2502

File: 07.Server-installation/04.Production-installation-with-kubernetes/03.Mender-server/docs.md [hunk 2]
```diff
@@ -180,7 +180,7 @@ useradm:
...
```
Comment (oldgiova) [inline 07.Server-installation/04.Production-installation-with-kubernetes/03.Mender-server/docs.md:183]: Same as previous comment

File: 07.Server-installation/04.Production-installation-with-kubernetes/03.Mender-server/docs.md [hunk 3]
```diff
@@ -352,7 +352,7 @@ EOF
 You can now update the Helm Chart with the included ingress:
 <!--AUTOVERSION: "helm upgrade --install mender mender/mender -f mender-%.yml"/integration -->
 ```bash
-helm upgrade --install mender mender/mender -f mender-3.7.7.yml
+helm upgrade --install mender mender/mender -f mender-3.7.8.yml
```
Comment (oldgiova) [inline 07.Server-installation/04.Production-installation-with-kubernetes/03.Mender-server/docs.md:355]: Same as previous comment

**Review by oldgiova — CHANGES_REQUESTED** [review]
Mender Server v3.7 requires the helm chart from the 5.x branch

File: 10.Downloads/docs.md [hunk 4]
```diff
@@ -700,7 +700,7 @@ HOSTED_MENDER_PASSWORD=<yoursecurepassword>
 And download it with:
 <!--AUTOVERSION: "/mender-gateway_%-1"/mender-gateway "/mender-gateway/debian/%/"/mender-gateway -->
 ```bash
-wget --auth-no-challenge --user "$HOSTED_MENDER_EMAIL" --password "$HOSTED_MENDER_PASSWORD" https://downloads.customer.mender.io/content/hosted/mender-gateway/debian/1.2.0/mender-gateway_1.2.0-1%2Bdebian%2Bbuster_armhf.deb
+wget --auth-no-challenge --user "$HOSTED_MENDER_EMAIL" --password "$HOSTED_MENDER_PASSWORD" https://downloads.customer.mender.io/content/hosted/mender-gateway/debian/1.2.1/mender-gateway_1.2.1-1%2Bdebian%2Bbuster_armhf.deb
```
Comment (merlin-northern) [inline 10.Downloads/docs.md:703]: I checked all the links and all the commands, and all the files to download up until this point.
```sh
#wget --auth-no-challenge --user "$HOSTED_MENDER_EMAIL" --password "$HOSTED_MENDER_PASSWORD" https://downloads.customer.mender.io/content/hosted/mender-gateway/debian/1.2.1/mender-gateway_1.2.1-1%2Bdebian%2Bbuster_armhf.deb
--2024-12-04 12:36:22--  https://downloads.customer.mender.io/content/hosted/mender-gateway/debian/1.2.1/mender-gateway_1.2.1-1%2Bdebian%2Bbuster_armhf.deb
Resolving downloads.customer.mender.io (downloads.customer.mender.io)... 54.160.79.2, 52.23.64.157
Connecting to downloads.customer.mender.io (downloads.customer.mender.io)|54.160.79.2|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-12-04 12:36:24 ERROR 404: Not Found.
```

Comment (lluiscampos) [reply 10.Downloads/docs.md:703]: Thank you for checking!

We are missing in 3.7.x the removal of Buster, I'll follow-up on that independently.

File: 07.Server-installation/04.Production-installation-with-kubernetes/03.Mender-server/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 07.Server-installation/04.Production-installation-with-kubernetes/03.Mender-server/docs.md]: Fixed now!

File: 10.Downloads/docs.md (hunk 4 above)
Comment (lluiscampos) [reply 10.Downloads/docs.md]: Fixed here:
* https://github.com/mendersoftware/mender-docs/pull/2504

**Review by oldgiova — APPROVED** [review]

**Review by merlin-northern — APPROVED** [review]
great work man

**Pull request merged**